h/services/oauth_validator.py

```diff
@@ -6,12 +6,13 @@
 
 from h import models
 from h.models.auth_client import GrantType as AuthClientGrantType
+from h.security.permissions import Permission
 from h.services.oauth_provider import ACCESS_TOKEN_PREFIX, REFRESH_TOKEN_PREFIX
 from h.util.db import lru_cache_in_transaction
 from h.util.uri import render_url_template
 
 AUTHZ_CODE_TTL = datetime.timedelta(minutes=10)
-DEFAULT_SCOPES = ["annotation:read", "annotation:write"]
+DEFAULT_SCOPES = [Permission.Annotation.READ, Permission.Annotation.WRITE]
```

Comment:
I think we should leave this as `["annotation:read", "annotation:write"]`. I don't think these are Pyramid permissions. They're set as the default scopes for OAuth access tokens. But I don't see them actually being used for anything

Reply:
Oh I see what happened here. 

I first changed all the strings like `"read"` into `"group:read"` etc. I picked colon so as not to clash with lots of our route definitions like `group.read`. These must have already been like this and just got caught up in the replacement.

I didn't think there was anything in the code base which followed this naming convention.